pkgs/servers/sql/postgresql/ext/timescaledb.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
   version = "2.2.1";
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ postgresql openssl ];
+  buildInputs = [ postgresql openssl libkrb5 ];
```

Comment:
```suggestion
  buildInputs = [ postgresql openssl ] ++ lib.optional stdenv.isDarwin libkrb5;
```

Reply:
It's broken on linux too https://nix-cache.s3.amazonaws.com/log/pxkl9r8rgc6lvmcy55lj612apkmvqbfj-timescaledb-2.2.1.drv, cause https://github.com/NixOS/nixpkgs/pull/121071